<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TIMETABLE</class>
 <widget class="QDialog" name="TIMETABLE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1606</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1541</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
  background-color: #0D1B2A;
    border-bottom: 2px solid #2d5a8c;
}</string>
      </property>
      <widget class="QPushButton" name="prevPageBtn">
       <property name="geometry">
        <rect>
         <x>1220</x>
         <y>80</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #F5F5F5;
    color: #333333;
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    padding: 6px 12px;
    font-size: 12px;
}
QPushButton:hover {
    background-color: #E8E8E8;
    border: 1px solid #2d5a8c;
}
QPushButton:pressed {
    background-color: #D8D8D8;
}</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nextPageBtn">
       <property name="geometry">
        <rect>
         <x>1320</x>
         <y>80</y>
         <width>41</width>
         <height>29</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #F5F5F5;
    color: #333333;
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    padding: 6px 12px;
    font-size: 12px;
}
QPushButton:hover {
    background-color: #E8E8E8;
    border: 1px solid #2d5a8c;
}
QPushButton:pressed {
    background-color: #D8D8D8;
}</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="pageNumberLabel">
       <property name="geometry">
        <rect>
         <x>1270</x>
         <y>80</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #FFF;
    font-weight: bold;
    font-size: 14px;
    background-color: transparent;
}</string>
       </property>
       <property name="text">
        <string>1/1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="saveAsBtn">
       <property name="geometry">
        <rect>
         <x>1370</x>
         <y>80</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2d5a8c;
    color: #FFFFFF;
    border: none;
    border-radius: 4px;
    padding: 8px 24px;
    font-weight: bold;
 
}
QPushButton:hover {
    background-color: #3d6fa8;
}
QPushButton:pressed {
    background-color: #1d4a7c;
}</string>
       </property>
       <property name="text">
        <string>Save as</string>
       </property>
      </widget>
      <widget class="QLabel" name="titleLabel">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>20</y>
         <width>461</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #FFF;
    font: bold 32pt &quot;Segoe UI&quot;;
    background-color: transparent;
    padding: 15px 20px;
}</string>
       </property>
       <property name="text">
        <string>View Timetable</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>1541</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #0D1B2A;
border-bottom: 1px solid #E0E0E0;</string>
      </property>
      <widget class="QTableWidget" name="timetableTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1511</width>
         <height>380</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget {
    background-color: #0D1B2A;
    alternate-background-color: #253445;
    gridline-color: #2d5a8c;
    border: 1px solid #2d5a8c;
}

QTableWidget::item {
    padding: 8px;
    color: #FFFFFF;
    background-color: #1a2535;
    border: 1px solid #2d5a8c;
}

QTableWidget::item:selected {
    background-color: #2d5a8c;
}

QHeaderView::section {
    background-color: #253445;
    color: #FFFFFF;
    padding: 8px;
    border: 1px solid #2d5a8c;
    font-weight: bold;
    font-size: 10pt;
}</string>
       </property>
       <row>
        <property name="text">
         <string>Monday</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Tuesday</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Wednesday</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Thursday</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Friday</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Saturday</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Sunday</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>8.00am</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>9.00am</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>10.00am</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>11.00am</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>12.00am</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>5.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>6.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>7.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>8.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>9.00pm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>10.00pm</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>1541</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #0D1B2A;
border-bottom: 1px solid #E0E0E0;</string>
      </property>
      <widget class="QLabel" name="totalCourseLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #2d5a8c;
    font: bold 18pt &quot;Segoe UI&quot;;
    background-color: transparent;
    padding: 10px;
}</string>
       </property>
       <property name="text">
        <string>Total Course: 0</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalHoursLabel">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #2d5a8c;
    font: bold 18pt &quot;Segoe UI&quot;;
    background-color: transparent;
    padding: 10px;
}</string>
       </property>
       <property name="text">
        <string>Total Hours: 0</string>
       </property>
      </widget>
      <widget class="QLabel" name="conflictsLabel">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>10</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #2d5a8c;
    font: bold 18pt &quot;Segoe UI&quot;;
    background-color: transparent;
    padding: 10px;
}</string>
       </property>
       <property name="text">
        <string>Conflicts: 0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backBtn">
       <property name="geometry">
        <rect>
         <x>1170</x>
         <y>50</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4472C4;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 0px;
}
QPushButton:hover {
    background-color: #3A5BA8;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteBtn">
       <property name="geometry">
        <rect>
         <x>1340</x>
         <y>50</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4472C4;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 0px;
}
QPushButton:hover {
    background-color: #3A5BA8;
}</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
